.github/workflows/e2e.yml

```diff
@@ -34,5 +34,6 @@ jobs:
                   composer require wp-cli/i18n-command
                   npm run build
                   npm install jest --global
+                  WP_VERSION=5.6.2
```

Comment:
@samueljseay any particular reason you set the version here instead of in line `28`? under the `env` key.